core/render.go

```diff
@@ -255,20 +255,20 @@ func (sc *Scene) doRebuild() {
 	sc.layoutRenderScene()
 }
 
-// prefSize computes the preferred size of the scene based on current contents.
+// contentsSize computes the size of the scene based on current contents.
 // initSz is the initial size -- e.g., size of screen.
 // Used for auto-sizing windows.
-func (sc *Scene) prefSize(initSz image.Point) image.Point {
+func (sc *Scene) contentsSize(initSz image.Point) image.Point {
```

Comment:
`sizeToContent` is probably a better name.

Reply:
this function does _not_ change the size. it returns the size that fits the content. sizeToContent sounds like it changes the size. sizeOfContent would be more accurate, but contentsSize is the same thing and simpler.

Reply:
Why not `contentSize`? I don't really see what the `s` is doing; is it "contents size" or "content's size"? Isn't "content" already plural?